source-google-analytics-data-api/tests/snapshots/snapshots__discover__capture.stdout.json

```diff
@@ -4,6 +4,17 @@
       "$schema": "https://json-schema.org/draft-07/schema#",
```

Comment:
We should have capture snapshot output tests as well if at all possible.

Reply:
@williamhbaker Does this account posses a valid dashboard so i can do that?

Reply:
I'm not honestly sure what a dashboard is in the context of google analytics. I think you set up the test property, though? So if its possible to add some data to `431462875` and capture it + snapshot, that would be ideal. Otherwise it'll be tough to feel confident the new version works, if we haven't been able to test it.

Reply:
It is really hard as we need a real website with the hook inside the pages html.
We can try with other data sources, but cant put it inside the CI pipeline without a estuary version of it.

Reply:
@williamhbaker How do you want to proceed? Test locally and validate the data?

Reply:
My objective is to get confidence that the new connector outputs data in roughly the same way as the old connector. Since there aren't any major differences in the output schemas, that's one point of confidence.

I'm hearing that it is not practical to have automated CI output tests, and that's kind of a bummer but it is what it is.

It would certainly be nice to know that this new connector has some ability to capture data. If there's a way for you to test it locally and validate the data, that would be great, and you can just add a description of what you did here and we'll call it good based on that manual testing.

I'm imagining make a static HTML page and opening it in my browser locally that has the hook in it and using that to generate some data in google analytics. I don't know if that would actually work though. If it does, I'd wonder why we couldn't have that as a CI test, since I would think that once the data is in google analytics it would stay there for later test runs, but maybe that is a flawed assumption.